tensorflow/compiler/tf2xla/kernels/conv_op_helpers.cc

```diff
@@ -179,8 +179,9 @@ Status ConvBackpropComputeDimensionsV2XlaShapes(
 
 }  // anonymous namespace
 
-absl::Span<const DataType> GetXlaConvTypes() {
-  return {DT_FLOAT, DT_BFLOAT16, DT_HALF, DT_DOUBLE};
+std::vector<DataType> GetXlaConvTypes() {
+  std::vector<DataType> ret = {DT_FLOAT, DT_BFLOAT16, DT_HALF, DT_DOUBLE};
+  return std::move(ret);
```

Comment:
Did not notice this earlier, but `return std::move(blah);` is an [anti-pattern](https://stackoverflow.com/questions/44065808/returning-stdvector-with-stdmove/44066170), although technically correct.  I'll fix this internally.